src/observers/telemetryObserver.ts

```diff
@@ -32,6 +32,17 @@ export class TelemetryObserver {
             case EventType.BuildCacheSizeCalculated:
                 this.handleBuildCacheSize(<BuildCacheSizeCalculated>event);
                 break;
+            case EventType.DependencyInstallStarted:
+                this.handleDependencyInstallStarted(<DependencyInstallStarted>event);
+                break;
+            case EventType.DependencyInstallCompleted:
+                this.handleDependencyInstallCompleted(<DependencyInstallCompleted>event);
+                break;
+            case EventType.PackageInstallCompleted:
+                this.handlePackageInstallCompleted(<PackageInstallCompleted>event);
+                break;  
+            // TODO: Send Metric for event PackageInstallAttemptFailed
```

Comment:
can be added now

Reply:
The vscode-extension-telemetry doesn't support `trackMetric` now, I have create a PR to that repo: https://github.com/microsoft/vscode-extension-telemetry/pull/42